JFrog IDEA Plugin fails to load builds that hold a build number with special characters

**Description:**
The plugin fails to load builds that hold a build number with special characters.

**Steps to reproduce:**

1. Create a Maven build with build number that includes special characters:
 `jf rt mvn install --build-name="core" --build-number="feature/ECOM-55157:2.17.0-SNAPSHOT:7"`
2. Promote this build to Artifactory:
 `jf rt bp "core" "feature/ECOM-55157:2.17.0-SNAPSHOT:7"`
3. Navigate to the plugin CI tab and select the newly created build

**Expected behavior**
A successfully loaded build under the CI tab.

**Error in Event log:**
`Couldn't find build info object in cache for 'core:feature/ECOM-55157:2.17.0-SNAPSHOT/7'.
[ERROR] Failed to load build 'core:feature/ECOM-55157:2.17.0-SNAPSHOT/'. java.io.IOException: Couldn't find build info object in cache for 'core:feature/ECOM-55157:2.17.0-SNAPSHOT/7'. at com.jfrog.ide.common.ci.CiManagerBase.loadBuildTree(CiManagerBase.java:127) at com.jfrog.ide.idea.ci.CiManager.loadBuild(CiManager.java:132) at com.intellij.util.messages.impl.MessageBusImpl.invokeMethod(MessageBusImpl.java:645) at com.intellij.util.messages.impl.MessageBusImpl.invokeListener(MessageBusImpl.java:620) at com.intellij.util.messages.impl.MessageBusImpl.deliverMessage(MessageBusImpl.java:417) at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:390) at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:372) at com.intellij.util.messages.impl.MessageBusImpl.access$200(MessageBusImpl.java:33) at com.intellij.util.messages.impl.MessageBusImpl$MessagePublisher.invoke(MessageBusImpl.java:179) at com.sun.proxy.$Proxy252.update(Unknown Source) at com.jfrog.ide.idea.ci.CiManager.loadFirstBuild(CiManager.java:169) at com.jfrog.ide.idea.ci.CiManager$1.run(CiManager.java:88) at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:450) at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:117) at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsync$5(CoreProgressManager.java:510) at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:243) at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:183) at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:705) at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:647) at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:63) at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:170) at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:243) at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700) at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668) at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665) at java.base/java.security.AccessController.doPrivileged(Native Method) at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665) at java.base/java.lang.Thread.run(Thread.java:829)`


**Versions:**
- JFrog IDEA plugin version: 1.14.1
- Xray version: 3.43.3

--- a/src/main/java/com/jfrog/ide/idea/ci/CiManager.java
+++ b/src/main/java/com/jfrog/ide/idea/ci/CiManager.java
@@ -124,7 +124,7 @@ public class CiManager extends CiManagerBase implements Disposable {
      *
      * @param buildGeneralInfo - The build general info
      */
-    public void loadBuild(GeneralInfo buildGeneralInfo) {
+    public void loadBuild(BuildGeneralInfo buildGeneralInfo) {
         ComponentsTree componentsTree = CiComponentsTree.getInstance(project);
         componentsTree.reset();
         ProjectsMap.ProjectKey projectKey = null;
@@ -135,7 +135,7 @@ public class CiManager extends CiManagerBase implements Disposable {
                 componentsTree.addScanResults(project.getName(), buildTree);
                 projectKey = ProjectsMap.createKey(project.getName(), buildTree.getGeneralInfo());
             } catch (IOException | ParseException | IllegalArgumentException e) {
-                Logger.getInstance().error(String.format(LOAD_BUILD_FAIL_FMT, buildGeneralInfo.getArtifactId(), buildGeneralInfo.getVersion()), e);
+                Logger.getInstance().error(String.format(LOAD_BUILD_FAIL_FMT, buildGeneralInfo.getBuildName(), buildGeneralInfo.getBuildNumber()), e);
             }
         }
         MessageBus projectMessageBus = project.getMessageBus();

--- a/src/main/java/com/jfrog/ide/idea/events/BuildEvents.java
+++ b/src/main/java/com/jfrog/ide/idea/events/BuildEvents.java
@@ -12,7 +12,7 @@ public interface BuildEvents {
     Topic<BuildEvents> ON_SELECTED_BUILD = Topic.create("Build selected", BuildEvents.class);
 
     /**
-     * Called when the the selected build is modified.
+     * Called when the selected build is modified.
      */
     void update(BuildGeneralInfo generalInfo);
 }

